setup.py

```diff
@@ -1,30 +1,37 @@
 from setuptools import setup, find_packages
 from pathlib import Path
 
-# Use semantic versioning
-VERSION = "1.0.0"
-
+# Minimum python version. Remember to also change in github workflows if increased
 PY_VERSION_REQUIRED = ">=3.7"
+
 PACKAGES_REQUIRED = {
     "core": [
         "numpy",
         "scipy",
     ],
     "test":[
-        "pytest"
+        "pytest",
     ]
 }
 
-# Read README.md to use as long description in package metadata
 root_dir = Path(__file__).parent.absolute()
+
+# Read README.md to use as long description in package metadata
 with (root_dir / "README.md").open("rt") as readme_file:
     long_description = readme_file.read().strip()
 
+# Read semantic version number from _version.py
+version_file_namespace = {} # type:ignore
+with (root_dir / "neffint" / "_version.py").open("r") as version_file:
+    exec(version_file.read(), version_file_namespace)
+version = version_file_namespace["__version__"]
+
 setup(
     name="neffint",
-    version=VERSION,
-    description="A python package for computing Fourier integrals using a Filon type method with non-equidistant grid spacing.",
+    version=version,
+    description="Computing Fourier integrals using a Filon type method with non-equidistant grid spacing.",
```

Comment:
I have to say I prefer the initial version of `description`... or is it that `A python package` is anyway printed by PyPI right before its description, or something like this?